code/parse/lua.cpp

```diff
@@ -3526,6 +3526,29 @@ ADE_FUNC(getFramesLeft, l_Texture, NULL, "Gets number of frames left, from handl
 	return ade_set_args(L, "i", num);
 }
 
+ADE_FUNC(getFrame, l_Texture, "number Elapsed time (secs), [boolean Loop]",
+		 "Get the frame number from the elapsed time of the animation"
+		 "The 1st argument is the time that has elapsed since the animation started"
+		 "If 2nd argument is set to true, the animation is expected to loop when the elapsed time exceeds the duration of a single playback",
+		 "integer",
+		 "Frame number")
```

Comment:
The documentation will look weird in the generated HTML because there are no spaces between the lines like here in the source code. If you intend to use newlines then you can use `<br>` at the end of each line.